### PR TITLE
[MAJOR] Connected app to MySQL database and setup Sequelize with Typescript

### PR DESCRIPTION
**Pull request by huangs08**

### CODE REVIEW

File: backend/src/main/config/config.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require('dotenv').config();
```
Comment (nasaku898) [inline backend/src/main/config/config.js:1]: is this required?

Comment (huangs08) [reply backend/src/main/config/config.js:1]: Yes, in the future we will add our own env variables

File: backend/package.json [hunk 2]
```diff
@@ -17,10 +17,15 @@
     "axios": "^0.21.3",
     "cors": "^2.8.5",
     "debug": "^4.3.2",
+    "dotenv": "^10.0.0",
     "express": "^4.17.1",
     "express-winston": "^4.2.0",
     "http-status-codes": "^2.1.4",
+    "mysql2": "^2.3.0",
```
Comment (nasaku898) [inline backend/package.json:24]: is this required ?

**Review by nasaku898 — CHANGES_REQUESTED** [review]
Overall looks good the implementation of the database. But could you look into your formatter and set it to prettier such that it will read the project formatter setting.

**Review by nasaku898 — APPROVED** [review]

File: backend/src/main/routes/AccountRoute.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    //   .patch((req: express.Request, res: express.Response) => {
+    //     res
+    //       .status(StatusCodes.OK)
+    //       .send(`PATCH requested for id ${req.params.accountEmail}`);
+    //   })
+    //   .delete((req: express.Request, res: express.Response) => {
+    //     res
+    //       .status(StatusCodes.OK)
+    //       .send(`DELETE requested for id ${req.params.accountEmail}`);
+    //   });
```
Comment (bowsong) [inline backend/src/main/routes/AccountRoute.ts:68]: Are these comments going to be uncommented in the future?

**Review by bowsong — DISMISSED** [review]

**Review by nasaku898 — APPROVED** [review]

**Review by nasaku898 — APPROVED** [review]

File: backend/src/main/routes/AccountRoute.ts (hunk 3 above)
Comment (nasaku898) [reply backend/src/main/routes/AccountRoute.ts:68]: Yeah it will be uncommented/implemented later

Comment (Manraj45) [reply backend/src/main/routes/AccountRoute.ts:68]: Yeah it will be uncommented/implemented later

Comment (huangs08) [reply backend/src/main/routes/AccountRoute.ts:68]: Yes, they will be uncommented later on when we implement the repositories, DTOs, and services.

**Review by bowsong — APPROVED** [review]

**Pull request merged**